<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::imageswindow</class>
 <widget class="QWidget" name="snowgui::imageswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QWidget" name="imageselectionWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="imageTree">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">4</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="snowgui::imagedetailwidget" name="imagedetailWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="snowgui::imagedisplaywidget" name="imageWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>snowgui::imagedisplaywidget</class>
   <extends>QWidget</extends>
   <header location="global">imagedisplaywidget.h</header>
   <container>1</container>
   <slots>
    <slot>rectangleSelected(astro::image::ImageRectangle)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>snowgui::imagedetailwidget</class>
   <extends>QWidget</extends>
   <header>imagedetailwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
